server/services/database/applicantProfile.js

```diff
@@ -63,12 +65,13 @@ const createApplicantProfile = ({
   city,
   cvLink,
   rightToWork,
+  profile_picture,
   userId
 }) => {
   return new Promise((resolve, reject) => {
     pool.query(
-      "INSERT INTO applicant_profile (name,about,city,cvLink,right_to_work,user_id ) VALUES($1,$2,$3,$4,$5,$6) RETURNING applicant_id",
-      [name, about, city, cvLink, rightToWork, userId],
+      "INSERT INTO applicant_profile (name,about,city,cvLink,right_to_work,profile_picture,user_id ) VALUES($1,$2,$3,$4,$5,$6,$7) RETURNING applicant_id",
+      [name, about, city, cvLink, rightToWork, profile_picture, userId],
```

Comment:
This one too should be: `profilePicLink`

Reply:
in the database part, I did it as profile_picture because it may be in the future something different so we don't need to change the names

Reply:
For the same reason, I mentioned at the top.

Reply:
done